### PR TITLE
Removing warnings about node/npm versions.

### PR DESCRIPTION
**Pull request by Blackbaud-BobbyEarl**
This could probably be worded better, but I'm waiting for builds and wanted to at least get the PR going.

### CODE REVIEW

File: src/app/learn/get-started/prereqs/initial-setup/index.html [hunk 1]
```diff
@@ -31,11 +28,8 @@
   </stache-page-anchor>
 
   <p>
-    Install NPM versions 3 or 4. NPM is the default JavaScript package manager for Node.js. As of Node.js version 0.6.3, it is bundled and installed automatically with the environment. We recommend versions 3 or 4 for their flat dependency structure. To check your version, run <stache-code>npm –v</stache-code> from the command line. To install a specific version, run <stache-code>npm install</stache-code> globally and specify the version. For example: <stache-code>npm install -g npm@4</stache-code>.
+    We require at least version 3 of NPM. NPM is the default JavaScript package manager for Node.js. As of Node.js version 0.6.3, it is bundled and installed automatically with the environment. We recommend versions 3 or 4 for their flat dependency structure. To check your version, run <stache-code>npm –v</stache-code> from the command line. To install a specific version, run <stache-code>npm install</stache-code> globally and specify the version. For example: <stache-code>npm install -g npm@4</stache-code>.
```
Comment (blackbaud-johnly) [inline src/app/learn/get-started/prereqs/initial-setup/index.html]: Do we still want the sentence "We recommend versions 3 or 4 for their flat dependency structure." My impression from talking to Steve was that we wanted to remove this line as well, but I might have misunderstood.

**Comment by blackbaud-johnly** [comment]
Resolves https://github.com/blackbaud/skyux2/issues/2159.

File: src/app/learn/get-started/prereqs/initial-setup/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/learn/get-started/prereqs/initial-setup/index.html]: @Blackbaud-BobbyEarl, do we want to keep that sentence about versions 3 or 4?

Comment (Blackbaud-BobbyEarl) [reply src/app/learn/get-started/prereqs/initial-setup/index.html]: Sorry, I might not be understanding, but 3 or 4 is no longer a requirement.  We still suggestion at least 3, but 5+ works too.

**Pull request merged**